app/src/main/java/org/simple/clinic/home/overdue/search/OverdueSearchScreen.kt

```diff
@@ -209,6 +221,27 @@ class OverdueSearchScreen : BaseScreen<
     )
   }
 
+  private fun hideKeyboardOnSearchResultsScroll(): Disposable {
+    return overdueSearchRecyclerView
+        .scrollStateChanges()
+        .filter { it == RecyclerView.SCROLL_STATE_DRAGGING }
+        .subscribe { hideKeyboard() }
+  }
```

Comment:
We also need to hide keyboard when search history is scrolled as well.

Reply:
https://obvious-hq.slack.com/archives/G018FHX0A5R/p1656063571310359